webpack.config.js

```diff
@@ -207,6 +213,27 @@ const DynamicCdnWebpackPluginConfig = {
           version: packageVersion,
           url: `https://unpkg.com/react-bootstrap@${packageVersion}/dist/react-bootstrap${packageSuffix}`,
         };
+      case '@mui/material':
+        return {
+          name: packageName,
+          var: 'MaterialUI',
+          version: packageVersion,
+          url: `https://unpkg.com/@mui/material@${packageVersion}/umd/material-ui.production${packageSuffix}`,
```

Comment:
Looks like they use a different naming convention here, so should be like:

`https://unpkg.com/@mui/material@${packageVersion}/umd/material-ui.${
            isProd ? 'production.min.js' : 'development.js'
          }`

Reply:
```suggestion
          url: `https://unpkg.com/@mui/material@${packageVersion}/umd/material-ui.${ isProd ? 'production.min.js' : 'development.js' }`,
```